uint16[] armCtr
uint16[] timeCtr
uint8 GripSta
